### ISSUE TITLE
WMI No values

### ISSUE THREAD

**Issue by aleohl** [LibreHardwareMonitor/LibreHardwareMonitor#1032]
Running Windows 11 Build 22H2 getting no value on WMI query `SELECT Identifier,Name,Value FROM Sensor`. Any ideas?
Found #581 but it seemed to work when @PhyxionNL tested, but it was a while ago.

**Comment by javguit** [comment]
Have you tried * instead of the specific fields?

**Comment by aleohl** [comment]
Yes, sir. I did try with `*`. I just checked with SimpleWMIView and now there is no items at all in WMI Class Sensor or Hardware. Strange.

**Comment by Slion** [comment]
To browse WMI I used that tool: https://github.com/vinaypamnani/wmie2
Tried the nightly build and it seems to be working.
